PWGHF/DataModel/CandidateReconstructionTables.h

```diff
@@ -54,6 +54,95 @@ using TracksPidPi = soa::Join<aod::pidTPCFullPi, aod::pidTOFFullPi>;
 using TracksPidKa = soa::Join<aod::pidTPCFullKa, aod::pidTOFFullKa>;
 using TracksPidPr = soa::Join<aod::pidTPCFullPr, aod::pidTOFFullPr>;
 
+using TracksPidTinyEl = soa::Join<aod::pidTPCEl, aod::pidTOFEl>;
+using TracksPidTinyMu = soa::Join<aod::pidTPCMu, aod::pidTOFMu>;
+using TracksPidTinyPi = soa::Join<aod::pidTPCPi, aod::pidTOFPi>;
+using TracksPidTinyKa = soa::Join<aod::pidTPCKa, aod::pidTOFKa>;
+using TracksPidTinyPr = soa::Join<aod::pidTPCPr, aod::pidTOFPr>;
+
+namespace pid_tpc_tof_utils
+{
+/// Function to combine TPC and TOF NSigma (for ML purposes)
+/// \param tpcNSigma is the (binned) NSigma separation in TPC (if tiny = true)
+/// \param tofNSigma is the (binned) NSigma separation in TOF (if tiny = true)
+/// \return Node containing the combined NSigma of TPC and TOF
+template <bool tiny = false, typename T1 = o2::framework::expressions::Node, typename T2>
+T1 combineNSigma(const T2& tpcNSigma, const T2& tofNSigma)
```

Comment:
What is the motivation for having the return type templated?

Reply:
Actually, it is a remnant of my tests during development (because I initially wrote this method to combine NSigma in a dynamic column in the reduced data model). I can remove it and directly impose the type as `o2::framework::expressions::Node` if you prefer.

Reply:
If there is currently no use case for having the template, it is indeed preferable to return an explicit type. Thanks